### PR TITLE
GTasks, TeamViewer Chat, APKMirror fix

### PR DESCRIPTION
**Pull request by Madis0**
#3 again

### CODE REVIEW

File: lite-apps/TeamViewer Chat/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+   "manifest_url":"https://hermit.chimbori.com/lite-apps/TeamViewer%20Chat.hermit",
+   "theme_color":"#2196f3",
+   "secondary_color":"#1976d2",
+   "icons":[
+      {
+         "src":"icon.png"
+      }
+   ],
+  "hermit_settings": {
+    "open_links": "browser"
```
Comment (manastungare) [inline lite-apps/TeamViewer Chat/manifest.json]: Why is this needed?

File: lite-apps/Google Tasks/manifest.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+   "manifest_version":1,
```
Comment (manastungare) [inline lite-apps/Google Tasks/manifest.json]: Could you please run these through a standard JSON formatter with indent set to two spaces? I’ll incorporate that into the preprocess scripts later, so this won’t need to be done manually.

I usually do:
`jq . manifest.json > tmp.json && mv tmp.json manifest.json`

File: lite-apps/TeamViewer Chat/manifest.json (hunk 1 above)
Comment (Madis0) [inline lite-apps/TeamViewer Chat/manifest.json]: Was thinking it would be nice to not disrupt chat flow and open links in a browser instead.

File: lite-apps/APK Mirror/manifest.json [hunk 3]
```diff
@@ -1,49 +1,49 @@
 {
-  "manifest_version": 1,
```
Comment (manastungare) [inline lite-apps/APK Mirror/manifest.json]: This file was actually correctly indented to begin with. OK, no problem, I’ll just add a formatter to reindent, so the tests will take care of it.
